docs/pages/concepts/offloading/mutating-webhook.md

```diff
@@ -0,0 +1,91 @@
+* The second `NodeSelectorTerm` enables Pod scheduling on all physical nodes, as they might not match the `ClusterSelector`.
+
+Hence, the resulting `NodeAffinity` constraint applied to a Pod would resemble the following:
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: liqo.io/region
+          operator: In
+          values:
+          - us-west-1
+      - matchExpressions:
+        - key: liqo.io/type
+          operator: NotIn
+          values:
+          - virtual-node
+```
+
+In case a given Pod already contains a set of `NodeAffinity` constraints, they are merged appropriately by the Liqo webhook with the ones above, ensuring that only the Nodes respecting the intersection of the restrictions can be selected as valid scheduling targets.
+
+#### Local PodOffloadingStrategy
+
+The `Local` strategy does not allow Pods to be scheduled on virtual nodes, hence behaving from this point of view as if Liqo were not present (although other resources, such as services, might be reflected to enable remote consumption).
+Consequently, in this case the webhook is not activated, and Pods are neither appended the *toleration* nor any additional *NodeAffinity* constraints.
+
+#### Remote PodOffloadingStrategy
+
+The `Remote` strategy forces Pods to be scheduled on virtual nodes only.
+Hence, the webhook adds the *toleration* to the virtual nodes *taint*, as well as mutates the pod affinity introducing a single `NodeSelectorTerm` with two (ANDed) `MatchExpressions`:
+
+* The first `MatchExpression` term enforces the constraints specified through the `ClusterSelector`.
+* The second `MatchExpression` ensures Pods are scheduled on virtual nodes only (i.e., with the `liqo.io/type=virtual-node` label).
+
+The resulting `NodeAffinity` constraint applied to a Pod resembles the following (in case it already contained a set of constraints, they would be merged accordingly):
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: liqo.io/region
+          operator: In
+          values:
+          - us-west-1
+        - key: liqo.io/type
+          operator: In
+          values:
+          - virtual-node
+```
+
+### Constraints violation
+
+The `NamespaceOffloading` resource specifies a set of constraints that applies to all pods of that namespace.
+Additionally, users can define additional and more specific requirements at the Pod level (i.e., through the `NodeAffinity` configuration), automatically merged by the Liqo webhook with the ones originating from the `NamespaceOffloading` resource.
+Still, care must be pay to prevent conflicting requirements, which would cause the Pod to remain in `Pending` status as unschedulable, as seen in the violation constraints example of the [Liqo Extended tutorial](/gettingstarted/extended/hard_constraints/).
```

Comment:
```suggestion
However, be careful about preventing conflicting requirements, which would cause the Pod to remain in `Pending` status as unschedulable, as seen in the violation constraints example of the [Liqo Extended tutorial](/gettingstarted/extended/hard_constraints/).
```